main.go

```diff
@@ -17,131 +14,13 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 
 	cobra "github.com/spf13/cobra"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	kvcorev1 "kubevirt.io/api/core/v1"
-	v1beta1 "kubevirt.io/api/export/v1beta1"
-	kubecli "kubevirt.io/client-go/kubecli"
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
 const (
-	diskPath             string = "./tmp/disk.img.gz"
-	diskPathDecompressed string = "./tmp/disk.img"
-	diskPathConverted    string = "./tmp/disk.qcow2"
+	diskPath string = "/tmp/targetpvc/disk.img"
```

Comment:
Will it be the disk path in all PVCs? or what matters here is just `disk.img`? If yes, is `disk.img` is always, and should we consider other formats? qcow2, compressed, etc.